src/Block/Service/AbstractAdminBlockService.php

```diff
@@ -19,8 +19,16 @@
 use Sonata\CoreBundle\Validator\ErrorElement;
 use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'AbstractAdminBlockService class is deprecated since 3.2x '.
```

Comment:
I remember me leaving this suggestion in a previous review :thinking: 
```suggestion
    'The '.__NAMESPACE__.'\AbstractAdminBlockService class is deprecated since sonata-project/block-bundle 3.x '.
```